app/controllers/TeamBinary.sublime-project

```diff
@@ -0,0 +1,8 @@
+{
+	"folders":
+	[
+		{
+			"path": "/Users/linatorres/Documents/Team-Binary-2015"
+		}
+	]
+}
```

Comment:
This file is specific t your text editor and shouldn't really be here, if need to keep this file, yo can add it to the the `.gitignore` file.